bundle/config/mutator/process_root_includes.go

```diff
@@ -49,6 +49,10 @@ func (m *processRootIncludes) Apply(ctx context.Context, b *bundle.Bundle) error
 			return err
 		}
 
+		if len(matches) == 0 {
+			return fmt.Errorf("%s defined in 'include' section does not match any files", entry)
+		}
+
```

Comment:
If the pattern equals `*.yml` and it doesn't lead to matches (after removing the seen ones, see block below), then it should not be an error. It is an error if the pattern is not a glob and the user wants to include a specific file.